README.md

```diff
@@ -112,6 +112,20 @@ You can group different conditions that gives flexibility in getting data from r
 and other(group and non-group condition) you can pass 'boolean' parameter in the same level as other conditions.*
 
 **Example:**`... ('field1 <> 'value1' or 'field2' > 1) or ('field3' in (1, 2) and 'field4' not in (1, 2)) ...`  
+- Relation existence criterion:  
+You can build queries to check on existence any model relations
+```php
+    $criteria = [
+        new RelationCriterion([
```

Comment:
Too verbose syntax :unamused: 

IMHO, would be better something like 
```php
new RelationCriterion('roles', ['slug', 'in', [1,2]], 'or' /* default = 'and' ? */);
```

Reply:
> Too verbose syntax 
> 
> IMHO, would be better something like
> 
> ```
> new RelationCriterion('roles', ['slug', 'in', [1,2]], 'or' /* default = 'and' ? */);
> ```

Maybe, but in this way we should change `Criterion` too, but it could be already use with this syntax. I believe that it change not to minor version